utils/src/main/kotlin/storage/FileArchiver.kt

```diff
@@ -73,8 +73,9 @@ class FileArchiver(
         zipFile.deleteOnExit()
 
         directory.packZip(zipFile, overwrite = true) { path ->
-            val relativePath = directory.toPath().relativize(path)
-            matcher.matches(relativePath.toString()).also { result ->
+            val relativePath = directory.toPath().relativize(path).toFile().invariantSeparatorsPath
```

Comment:
This way only one conversion is needed:

    val relativePath = path.toFile().relativeTo(directory).invariantSeparatorsPath

Reply:
Done.